router/router.go

```diff
@@ -127,10 +128,7 @@ func (rt *HandleT) workerProcess(worker *workerT) {
 		batchTimeStat.Start()
 		logger.Debugf("Router :: trying to send payload to %s. Payload: ", rt.destID, job.EventPayload)
 
-		userID := integrations.GetUserIDFromTransformerResponse(job.EventPayload)
-		if userID == "" {
-			panic(fmt.Errorf("userID is empty"))
-		}
+		userID, canEventBeMappedToUser := integrations.GetUserIDFromTransformerResponse(job, job.EventPayload)
```

Comment:
Let's remove the dependency between integrations and jobsdb.

Change it to something like following?

```
if job.UserID != "" {
   userID = job.UserID
} else {
   userID = integrations.GetUserIDFromTransformerResponse(job.EventPayload)
}
```